commit 29efa8fc185e0fab049d8fa0403d77e14a21ab1e (refs/changes/16/5416/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2019-01-23T13:50:02-08:00 (9 months ago)
    
    ADMINUI-2411 Admin portal refers to kvm instead of HVM for packages

diff --git a/www/js/tpl/packages-form.hbs b/www/js/tpl/packages-form.hbs
index 1cfb905e..5f0a9297 100644
--- a/www/js/tpl/packages-form.hbs
+++ b/www/js/tpl/packages-form.hbs
@@ -154,9 +154,9 @@
               class="form-control"
               id="package-vcpus"
               name="vcpus" placeholder="cpus">
-          <p class="help-block">Applies to KVMs only.
+          <p class="help-block">Applies to HVMs only.
             <a data-toggle="tooltip" data-placement="bottom" data-trigger="hover"
-              title="Number of virtual CPUs presented to the guest OS. Required when creating a KVM.">
+              title="Number of virtual CPUs presented to the guest OS. Required when creating a HVM.">
               What is this?</a>
           </p>
         </div>
